docs/user-guide-state-reporting.md

```diff
@@ -67,6 +67,10 @@ spec:
     - name: dbus-socket
       mountPath: /run/dbus/system_bus_socket
     env:
+    - name: POD_NODE_NAME
+      valueFrom:
+        fieldRef:
+            fieldPath: spec.nodeName
```

Comment:
different indentation :(

Reply:
We need to generate this part of docs from templates, but not here.

Reply:
You are right. Or simply just reference generated manifests from docs.